File: backend_old/mysql/table/companyBanch.go
package table

import (
	"time"

)

//公司部們
type CompanyBanchTable struct {
	Id int64 `json:"Id"`
	CompanyId int64 `json:"CompanyId"` // 公司編號
	BanchName string `json:"BanchName"` // 公司部們名稱
	BanchShiftStyle string `json:"BanchShiftStyle"` // 部門班表樣式
	CreateTime time.Time `json:"CreateTime"`//創建的時間
	LastModify time.Time `json:"LastModify"`// 上次修改的時間
	UserTotal int // 總人數
}

File: frontend_web/src/types/companyBanch.ts
declare namespace companyBanchTypes {
    interface TABLE {
        CompanyId: number
        BanchId: number
        BanchName: string
        Sort: number
        DeleteFlag: Flag
        DeleteTime: string
        CreateTime: string
        LastModify: string
    }

    interface reducerType {
        all: TABLE[]
        selector: TABLE[]
    }
}
export default companyBanchTypes
